styleguide/source/_patterns/04-templates/three-column.twig

```diff
@@ -0,0 +1,38 @@
+{% block skipNav %}
+  {% include "@atoms/skip-nav.twig" %}
+{% endblock %}
+
+<div class="container">
+  {% block ribbon %}
+    {% include '@base/placeholder.twig' with {'placeholder':{'text':'Topic page  ribbon'}} %}
+  {% endblock %}
+
+  {% block header %}
+    {% include '@base/placeholder.twig' with {'placeholder':{'text':'Topic page header'}} %}
```

Comment:
Does it make sense to use `with` here to pass variables, given that all data is supposed to come from .json?

Reply:
I replace the includes with a json file